packages/main/src/App.js

```diff
@@ -38,14 +38,14 @@ const ProtectedRoute = ({ children, ...rest }) => {
     <Route
       {...rest}
       render={({ location }) =>
-        auth.user ? (
+        auth.user !== true ? (
           children
         ) : location.pathname === "/signout" ? (
           children
         ) : (
           <Redirect
             to={{
-              pathname: "/login",
+              pathname: "/signout",
```

Comment:
Same to this

Reply:
Apologies. Those changes were made during the initial Zuri chat task that involved the teams. Had the environment up and running since then. Do I need to undo?

Reply:
Yes. If they're not involved in this or then you'll have to remove them from the commits in this branch